views/sponsors.php

```diff
@@ -27,6 +29,124 @@
+            </a>
+            <a href="http://www.7pcb.com/" target="_blank">
+                <img style="width: 160px; margin: 20px 20px;" src="../public/images/company-logos/bittele-logo.png">
+            </a>
+            <a href="http://www.reverie.com/" target="_blank">
+                <img style="width: 100px; margin: 20px 20px;" src="../public/images/company-logos/reverie.jpg">
+            </a>
+            <a href="https://credibly.com/" target="_blank">
+                <img style="width: 200px; margin: 20px 20px;" src="../public/images/company-logos/credibly.jpg">
+            </a>
+            <a href="https://www.greektowncasino.com/" target="_blank">
+                <img style="width: 140px; margin: 20px 20px;" src="../public/images/company-logos/greektown.png">
+            </a>
+            <a href="https://www.vividseats.com/" target="_blank">
+                <img style="width: 170px; margin: 20px 20px;" src="../public/images/company-logos/vivid-seats-logo.png">
+            </a>
+        </div>
+        <div class = "tiersbox" id = "label"><h1>SILVER</h1></div>
+</div>
+</div>
+<h1 id = "titles" align = "left" style = "margin-top: 30px; margin-bottom: 20px">
+  <span class="black">BENEFITS OF</span>
+  <span class="ieee-blue">SPONSORING</span>
+</h1>
+<ul>
+    <li>Priority status and multiple events (info-sessions, tech-talks, etc.) per year (as per package)</li>
+   <li>Priority access to award winning event organization, management and publicity</li>
+   <li>Advertising your job/internship openings through us</li>
+   <li>Reaching out to 1000+ mailing list members and 400+ club members for publicity</li>
+   <li>Getting involved with one of the largest engineering student groups in UCLA</li>
+   <li>Being advertised at various state-wide engineering competitions</li>
+   <li>Being a generous contributor to the development of young engineers</li>
+   <li>Getting all this for free! <b>Donations to us are tax-deductible</b></li>
+    </ul>
+    <br>
+<h1 id = "titles" align = "left" style = "margin-top: 40px; margin-bottom: 20px">
+  <span class="black">BECOMING A</span>
+  <span class="ieee-blue">SPONSOR</span>
+</h1>
+    <div class = "tierscontainer">
+        <div class = "tiersbox" id = "box" style = "float: left">
+            <h1 id = "tier">SILVER</h1>
+            <p id = "price">$500 - $999</p>
+        </div> 
+        <div class = "tiersbox" id = "box" style = "float: left">
+            <h1 id = "tier">GOLD</h1>
+            <p id = "price">$1000 - $1999</p>
+        </div>
+        <div class = "tiersbox" id = "box" style = "float: left">
+            <h1 id = "tier">PLATINUM</h1>
+            <p id = "price">$2000 - $4999</p>
+        </div>
+        <div class = "tiersbox" id = "box" style = "float: left">
+            <h1 id = "tier">DIAMOND</h1>
+            <p id = "price">$5000</p>
+        </div>
+    </div>
+<div class = "interestbox">
+<p style = "line-height: 29px">
```

Comment:
I think you might wanna try using "line-height: 1.5" or "line-height: 2" which represents 1.5 spacing or double spacing. But if you're ok with 29px, then that's cool too.